openedx/features/qverse_features/registration/signals.py

```diff
@@ -16,13 +18,14 @@
 @receiver(post_save, sender=BulkUserRegistration)
 def create_users_from_csv_file(sender, instance, created, **kwargs):
     """
-    Creates/Updates users and their profiles.
+    Creates/Updates users and their profiles and sends registration emails.
```

Comment:
```suggestion
    Creates/Updates users, their profiles and sends registration emails.
```

Reply:
What Umar wrote is correct too actually 😅